oeps/architectural-decisions/oep-0064-mobile-application-codebase-modernization.rst

```diff
@@ -0,0 +1,103 @@
+
+Motivation
+**********
+
+The motivation for this shift comes from an opportunistic review of newly rebuilt mobile applications from Raccoon Gang that seek to remove some of the mobile contribution impediments that the existing applications have. The original mobile application repositories were authored in 2015 and have progressively started including new architectural patterns and the use of Swift / Kotlin, but challenges remain for contribution and coded clarity. Additionally, coding patterns, libraries, and architectural patterns have changed significantly since the original authorship of our mobile applications, and the newly rebuilt Raccoon Gang applications are open sourced and a great starting point for renewed emphasis on mobile learning options for Open edX platform adopters and learners.
+
+Additionally, Axim Collaborative is looking to expand the way the Open edX mobile applications can serve as a tool for expanded access to learning opportunities. Mobile application learning is a key requirement to modern learning platforms and we believe that with concentrated effort new contributions and improvements in the mobile space will help expand the use of the open edx platform.
+
+
+Specification 
+************* 
+
+The existing mobile repositories are maintained by edX / 2U and can be accessed here:
+
+* https://github.com/openedx/edx-app-ios
+* https://github.com/openedx/edx-app-android
+
+* Support for the existing mobile applications will continue through the Redwood release. Starting with the release after Palm, we will designate the new repositories as the default Open edX mobile applications to encourage adoption and use of the new applications for community team members.
+* Once available, we will link to documentation within the new repositories to describe transition options for community members migrating from existing builds of the older codebases.
+* After the publication of release after Palm we will update the original repositories with the suffix ‘archived’ to help make clear that these are no longer the default mobile application builds moving forward. A corresponding DEPR ticket linked to this OEP once it exists. 
+
+The new mobile repositories are maintained by Raccoon Gang and can be accessed here:
+* https://github.com/raccoongang/educationx-app-ios
+* https://github.com/raccoongang/educationx-app-android
+
+* The new repositories will be migrated to the openedx github organization under the name openedx-app-ios and openedx-app-android.
+* The new repositories will be maintained by the Racoon Gang team with Marco Morales serving as the product lead for the project.
```

Comment:
Will this be a maintainership as per oep-55 and oep-54?

Reply:
I don't think maintainers should be named in an OEP. It will be easy to forget to update. This information should live in `catalog-info.yml`

Reply:
I suggest striking this line entirely. If you want to call out you can say `The new repositories will be maintained in accordance with OEP-55`